test/rubygems/test_gem_installer.rb

```diff
@@ -1142,14 +1142,24 @@ def test_install_extension_dir_is_removed_on_reinstall
   end
 
   def test_find_lib_file_after_install
```

Comment:
I see. but this test contained ruby core workaround is not working <= 1.9.2. We need to skip these versions same as `test_install_extension_flat`